clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -242,6 +251,15 @@ private ResourcePageSource createPageSource(String poolName, ServerSideConfigura
     return pageSource;
   }
 
+  private void registerStatistic(Object context, String alias, String observerName, Callable callable) {
+    Set<String> tags = new HashSet<String>(Arrays.asList("ClusteredServerSide","tier"));
```

Comment:
should replace ClusteredServerSide by `PageSource ` : and put a constant, like it is done for observers on client side.